.golangci.yml

```diff
@@ -118,10 +116,10 @@ linters-settings:
       - name: defer
 
       - name: argument-limit
-        arguments: 6
```

Comment:
do you have the latest golangci version? `1.64.7`
Initially, I had list there, and it complained that it should it expects integer rather than an array